README.md

```diff
@@ -1,18 +1,49 @@
 # vets-api-clients
 
-This repository contains resources for accessing vets-api as a third-party
-client. It includes API specifications, sample clients, and general
-access information.
+Welcome to the public-facing repository for **VA Lighthouse APIs**.  For the most up-to-date API documentation, please visit [developer.va.gov/explore](https://developer.va.gov/explore).
 
-**Third-party access to vets-api is currently in limited beta status, and
-is only available in development environments.**
+If you would like to improve or suggest **edits to API documentation** for the VA APIs, please visit [this repo](https://github.com/department-of-veterans-affairs/developer-portal/tree/master/src/content/apiDocs).
+
+**Materials in this repo may be outdated as of July 2019.** This repository contains resources for accessing vets-api as a third-party client. It includes API specifications, sample clients, and general access information.   Anything older than April 2019 may be considered deprecated.  Again, please refer to the [developer portal](https://developer.va.gov/explore) linked above for current docs.
+
+Third-party access to vets-api is available to anyone with a developer token (API key), which can be obtained via a [quick application form](https://developer.va.gov/apply).  Production access will be granted later by scheduling a [demo of your application](https://developer.va.gov/go-live) for stakeholders.
+
+Note that the Health (FHIR / Argonaut) APIs for access to veteran health records are not yet in production; dev access in sandbox using mock patient data is available, and we expect these APIs to be in production by fall of 2019.
```

Comment:
Veteran - capitalize V in "access to veteran health records..."

Reply:
done